src/domain/event/__tests__/CreateEventPage.test.tsx

```diff
@@ -40,7 +41,7 @@ import CreateEventPage from '../CreateEventPage';
 expect.extend(toHaveNoViolations);
 
 beforeEach(() => {
-  jest.spyOn(Router, 'useHistory').mockReturnValue({});
+  jest.spyOn(Router, 'useHistory').mockReturnValue({} as any);
```

Comment:
Maybe you could use render from testUtils to get rid of router mocks and apollo mocked provider